stage1/stage1.mk

```diff
@@ -66,7 +66,7 @@ $(foreach f,$(STAGE1_FLAVORS), \
 	$(eval STAGE1_SECONDARY_STAMPS_$f :=) \
 	$(eval STAGE1_ACIDIR_$f := $(BUILDDIR)/aci-for-$f-flavor) \
 	$(eval STAGE1_ACIROOTFSDIR_$f := $(STAGE1_ACIDIR_$f)/rootfs) \
-	$(eval STAGE1_ACI_IMAGE_$f := $(BINDIR)/stage1-$f.aci) \
+	$(eval STAGE1_ACI_IMAGE_$f := $(TARGET_BINDIR)/stage1-$f.aci) \
```

Comment:
In the same file, there is a rule that begins with `$$(STAGE1_ACI_IMAGE_$1): $$(ACTOOL_STAMP) |`. You should update the directory there too to `$(TARGET_BINDIR)`.